lib/spi/addon-contracts.ts

```diff
@@ -1,10 +1,13 @@
 import { Team, ClusterInfo } from '.';
+import { Construct } from '@aws-cdk/core';
 
 /**
  * ClusterAddOn is the interface to which all Cluster addons should conform.
+ * @param clusterInfo
+ * @returns Promise<cdk.Construct>
```

Comment:
It make sense to add that if implementor does not return the promise, then no dependency features will apply.

Reply:
added